app/components/windows/go-live/GoLiveChecklist.tsx

```diff
@@ -158,16 +153,25 @@ export default class GoLiveChecklist extends TsxComponent<Props> {
       >
         <CheckMark state={state} />
         <span>{title}</span>
-        {renderYTPercentage && this.renderYoutubePercentage()}
         {renderStreamDelay && <span class={styles.pending}> {this.delaySecondsRemaining}s</span>}
       </li>
     );
   }
 
-  private renderYoutubePercentage() {
-    if (this.view.info.checklist.publishYoutubeBroadcast === 'not-started') return '';
-    const progressInfo = this.youtubeService.progressInfo;
-    return <span class={styles.pending}> {progressInfo.progress * 100}%</span>;
+  private renderWarning() {
+    if (!this.view.info.warning) return;
+    return (
+      <div class="section selectable">
+        <p>
+          {$t(
+            'Auto-start is disabled for you broadcast. You should manually publish you stream in the Youtube Studio',
```

Comment:
Grammatical fixes:
`Auto-start is disabled for your broadcast. You should manually publish your stream from Youtube Studio`

Reply:
Also, just to confirm - we are trying to set Auto-start on the broadcast from the API, so this should rarely/never happen, right?

Reply:
It happens only when you explicitly disable `Auto-start` from SLOBS.

Reply:
Got it - sounds good.  Thanks!